thrift-gen/agent/ttypes.go

```diff
@@ -1,21 +0,0 @@
-// Autogenerated by Thrift Compiler (0.9.3)
```

Comment:
why are there so many files removed? Is the content migrated elsewhere?

Reply:
I have removed entire `thrift-gen` directory and run the generator. Maybe the new generator uses a different file structure?